src/settings.ts

```diff
@@ -116,6 +110,10 @@ export function legacySettings(): Map<string, Object> {
   let settings: Map<string, Object> = new Map<string, Object>();
   let value: Object = undefined;
 
+  // puppet.editorService.docker.imageName
```

Comment:
The legacy stuff below is for redirecting old settings to their new location.  As this is a removal, there's no need to have this here (There's nothing to redirect to)